windows/win_msi.py

```diff
@@ -45,6 +45,16 @@
         description:
             - Path to a file created by installing the MSI to prevent from
               attempting to reinstall the package on every run
+    wait:
+        version_added: "2.0"
+        description:
+            - Specify whether to wait for install or uninstall to complete before continuing.
+        choices:
```

Comment:
I think this is just reflecting the rest of ansible - the various ways of specifying booleans are mentioned here, for example: http://docs.ansible.com/ansible/YAMLSyntax.html